content/24.vaccines-novel.md

```diff
@@ -254,7 +245,7 @@ Both the innate and adaptive immune responses can potentially respond to the vec
 Different vectors are associated with different levels of reactogenicity; for example, adenoviruses elicit a much stronger innate immune response than replication deficient adeno-associated viruses derived from parvoviruses [@doi:10.1038/gt.2009.148].
 Additionally, using a virus circulating widely in human populations as a vector presents additional challenges because vaccine recipients may already have developed an immune response to the vector [@doi:10.3390/vaccines2030624]. <!-- So does this also have implications about using the same vector multiple times to develop different vaccines? I recall reading about about that but not the details. -->
```

Comment:
I'm not sure and I'm not finding anything with the search terms I can think of! @RLordan, any ideas on this?

Reply:
Yes, this is ringing bells. Came up for the sputnik vaccine, its why they used two different viral vectors... There is some population immunity to commonly used vectors. See example for H5N1 vaccine (10.1371/journal.pone. 0033428) and these papers (https://doi.org/10.1016/j.vaccine.2011.05.025) & (https://doi.org/10.1126/sciimmunol.abi8635)